src/main/resources/swagger/cromwell.yaml

```diff
@@ -707,6 +714,11 @@ definitions:
       jobId:
         type: string
         description: Backend-specific job ID
+      failures:
+        type: object
+        schema:
+          $ref: '#/definitions/FailureMessage'
+        description: If the call failed, the set of recorded failure messages for the call.
```

Comment:
Not merge-blocking- but I'm not sure the convention for the end of descriptions. Some end in `.`, others don't?